### ISSUE TITLE
Dark mode feature

### ISSUE THREAD

**Issue by max-programming** [dubinc/dub#56]
This project is super cool, and I started using it instead of bitly. Honestly, it's faster and has better UX!

It would be great if it has a dark mode feature and a toggle for that. 

I would be happy to take that challenge to implement the dark mode feature

**Comment by steven-tey** [comment]
Yes! This would be AMAZING! I actually worked on this but decided to rip it out before launch because I was running low on time and it wasn't the top priority...happy to collaborate on the PR! :)

**Comment by max-programming** [comment]
@steven-tey  Cool! Assign it to me!

**Comment by brandonmcconnell** [comment]
+1

Would love to see dark mode added eventually, as well

**Comment by steven-tey** [comment]
Re-opening this cause we're finally getting dark mode prioritized! 😁 

![Image](https://github.com/user-attachments/assets/3cfb93e7-11ce-40b7-8fb4-2f060fa6c98b)

**Comment by steven-tey** [comment]
Actually, combining this to #1669 👍
